docs/en/stack/ml/df-analytics/ml-trained-models.asciidoc

```diff
@@ -13,16 +13,223 @@ information about this process, see <<ml-supervised-workflow>> and
 {ref}/delete-trained-models.html[delete trained models].
+
+
+[discrete]
+[[move-between-clusters]]
+== Moving a trained model between clusters
+
+It is a common scenario that the {ml} models are trained in a cluster that is 
+used for development and test purposes, and they are used in another cluster 
+which is the production environment. In this case, you need to move your trained 
+model from one cluster to another. The trained model APIs enable you to move 
+your trained model between clusters. The following description shows you the 
+process step by step.
+
+1. (Optional) In the cluster where you trained the model, make the call below by 
+using the console in **Dev Tools** to get the configuration information of your 
+trained models.
++
+--
+
+[source,console]
+--------------------------------------------------
+GET _ml/inference/
+--------------------------------------------------
+// TEST[skip:setup kibana sample data]
+
+The API response contains the `model_id` of the trained models. Check the 
+`model_id` of the trained model you want to move, you need to add it to the API 
+call in the next step.
+--
+
+2. Use the {ref}/get-inference.html[GET trained model API] to get the trained 
+model definition. You need to specify the following query parameters in the 
+call:
++
+--
+* `for_export`: This parameter allows the model to be in an acceptable format to 
+be retrieved and then added to another cluster. Set it to `true`.
+
+* `include_model_definition`: It specifies whether the model definition is 
+returned in the response. Set it to `true`.
+
+* `decompress_definition`: It specifies in what format the included model 
+definition should be returned. Set it to `false` for getting a custom compressed 
+format. It is also valid to use the JSON format, but it is not optimal. As the 
+decompressed definition may be significantly larger, it is recommended to use 
+the compressed format.
+   
+The following call is an example to get the trained model definition. (Replace 
+`<model_id>` with the actual ID of the trained model.)
+
+[source,console]
+--------------------------------------------------
+GET _ml/inference/<model_id>?for_export=true&include_model_definition=true&decompress_definition=false
+--------------------------------------------------
+// TEST[skip:setup kibana sample data]
+
+The API response returns a `trained_model_configs` array that contains a 
+`compressed_definition` object and the analytics and inference configuration 
+information.
+--
+
+3. Copy the content of `trained_model_configs`.
+
+4. Use the {ref}/put-inference.html[Create trained model API] in the cluster you 
+want to move the trained model to. Paste the content of the 
+`trained_model_configs` to the request body of the API call.
++
+--
+The following call is an example to create the trained model. The actual 
+`compressed_definition` value is replaced by a place holder.
+
+.Create trained model API example
```

Comment:
Looking at this example, it made me wonder which of these options were required to be the same as in the original cluster (other than compressed_definition).  Do they all have to be identical?  If not, I'm not sure if there's a lot of value to having this example and wonder if it should be removed, since the only option of importance is not shown fully anyway.

Reply:
@lcawl AFAIK, you can change the metadata in the PUT body, but it does not affect your model. However, the model configuration data will be inaccurate in this case as the model hasn't been trained with that configuration. The compressed_definition is a really long (in this example it's almost 50.000 character-long) series of characters and it is not human-readable. This is why I would omit it. I'll ask Ben about his opinion when I open the PR for review.

Reply:
I agree with not fully expanding the `compressed_definition`, but it might be useful to elide the rest of the config for brevity.

Reply:
I removed the API example in 106e319.